src/Uno.UI/Extensions/ViewHelper.iOSmacOS.cs

```diff
@@ -27,15 +28,19 @@ namespace Uno.UI
 	public static class ViewHelper
 	{
 #if __IOS__
+		[Obsolete("This return the value from the original screen. Use 'DisplayInformation.RawPixelsPerViewPixel' to get the value for the current screen.")]
```

Comment:
We generally don't mark fields as obsolete because it's breaking for warning-as-error-enabled builds. Let's mark those fields are non-browsable for now, and we can introduce a breaking change in Uno 5 (https://github.com/unoplatform/uno/issues/8339).